integreat_cms/static/src/js/forms/icon-field.tsx

```diff
@@ -32,13 +32,13 @@ window.addEventListener("load", () => {
      */
     const setIcon = () => {
         // Show preview
-        document.getElementById("icon_preview").classList.remove("hidden");
+        document.getElementById("icon_preview")?.classList.remove("hidden");
         // Change icon label
-        document.getElementById("set-icon-label").classList.add("hidden");
-        document.getElementById("change-icon-label").classList.remove("hidden");
+        document.getElementById("set-icon-label")?.classList.add("hidden");
+        document.getElementById("change-icon-label")?.classList.remove("hidden");
         // Toggle remove/restore buttons
-        document.getElementById("restore-icon").classList.add("hidden");
-        document.getElementById("remove-icon").classList.remove("hidden");
+        document.getElementById("restore-icon")?.classList.add("hidden");
+        document.getElementById("remove-icon")?.classList.remove("hidden");
```

Comment:
This is the wrong way to go - the icon-field functionality should be completely independent from the poi map icon preview. Please use different ids instead of the `?` everywhere.

Reply:
They are now reverted.